internal/commands/interceptor/process.go

```diff
@@ -83,7 +83,7 @@ func parseProcess(required bool, idFlagName, nameFlagName, instanceIDFlagName st
 			if instanceID == "" {
 				return fmt.Errorf(`"--%s" is specified but its related service name or id is not given`, nameFlagName)
 			}
-			id = fmt.Sprintf("%x", sha256.New().Sum([]byte(fmt.Sprintf("%s_%s", instanceID, name))))
+			id = fmt.Sprintf("%x", sha256.Sum256([]byte(fmt.Sprintf("%s_%s", instanceID, name))))
```

Comment:
BTW @mrproliu , I checked how the OAP build the process id, it’s sha256(name_instanceId), but here, it’s sha256(instanceId_name), is it a bug?

https://github.com/apache/skywalking/blob/7dccf96f30ec8a02e75523f3174540fb0b8d4c8a/oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/IDManager.java#L318-L319